backoffice/src/views/exports/certificates/CertificateView.tsx

```diff
@@ -34,20 +34,35 @@ export const CertificateView: FunctionComponent<CertificateViewProps> = ({
   }
 };
 
-export const LetterView: FunctionComponent<CertificateViewProps> = ({
+interface LetterViewProps {
+  exportsGateway: IExportsGateway;
+  beaconId: string;
+  letterType: string;
+}
+
+export const LetterView: FunctionComponent<LetterViewProps> = ({
   exportsGateway,
   beaconId,
+  letterType,
 }): JSX.Element => {
   const [beacon, setBeacon] = useState<IBeaconExport>({} as IBeaconExport);
+  console.log(letterType);
 
   useEffect(() => {
     exportsGateway.getExportDataForBeacon(beaconId).then(setBeacon);
```

Comment:
ahh, my bad I fixed this on the bulk export branch. We need two endpoints, one for certs and one for letters. 

Are you happy for me to go in and sort this, I've written it just need to pull it across from another branch